src/App.js

```diff
@@ -16,7 +16,7 @@ const cities = [
   'Cuchilla Alta, uru',
   'Molina de Aragón, es',
   'Alaska, can',
-  'Brasilia, br',
+  'Sao paulo, br',
```

Comment:
En este caso Sao Paulo debería empezar con mayúsculas ambas palabras.
```suggestion
  'Sao Paulo, br',
```